scripts/docs-to-skills.py

```
@@ -1436,33 +1639,43 @@ def group_by_content_type(pages: list[DocPage]) -> dict[str, list[DocPage]]:
+        if doc_path.parent.name.startswith("_"):
             continue
         # Skip build artifacts
-        if "_build" in md_path.parts:
+        if "_build" in doc_path.parts:
             continue
 
         try:
-            page = parse_doc(md_path)
+            page = parse_doc(doc_path, doc_platform=doc_platform)
             pages.append(page)
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)